Helper_Scripts/demo_chat_sse_and_metrics.py

```
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)